app.rb

```diff
@@ -0,0 +1,34 @@
+require 'sinatra/base'
```

Comment:
As a general comment: I found your code very well thought out and put together. It is very easy to both understand and follow.

Reply:
I liked your use of docs and also external style sheets.